uint8 version       #0 for request, 1 for response
uint8 layer         # Layer data is from
uint16 position     # Key value offset
uint32 keyID        # KeyID of Configuration
string keyName      # Name of Configuration
uint64 data         # Configuration Data